django/contrib/postgres/functions.py

```diff
@@ -2,6 +2,13 @@
 
 
 class RandomUUID(Func):
+    """
+    Without `pgcrypto` or `uuid-ossp` extensions enabled, available only
+    since PostgreSQL 13.
+
+    https://www.postgresql.org/docs/13/functions-uuid.html
+    """
+
```

Comment:
Django 5.2 (currently under development) only supports PostgreSQL 14+, so this comment is unnecessary.

Reply:
https://github.com/django/django/pull/18365/commits/51f3d91cf72bc889ecff86fee0b7ba0f4084a826